README.md

```diff
@@ -1,135 +1,33 @@
-# Ubuntu Ceph Container Images
-Home to Ubuntu based Ceph container Images.
+# Ceph ROCKs
+Home to Ubuntu based Ceph OCI Images.
```

Comment:
Minor suggestion: could add a link to explain OCI, e.g. to https://en.wikipedia.org/wiki/Open_Container_Initiative , either here or below

Reply:
+1